.github/workflows/push.yml

```diff
@@ -49,6 +49,35 @@ jobs:
               composer install --no-progress
               composer rector
 
+  docs:
+      name: Documentation
+      runs-on: ubuntu-latest
+      steps:
+          - uses: actions/checkout@v2
+          - name: Setup PHP
+            uses: shivammathur/setup-php@v1
+            with:
+                php-version: 7.2
+                coverage: none # disable xdebug, pcov
+          - run: |
+              composer install --no-progress
+              composer check-docs
+              
+  simple-checks:
+      name: Simple checks
```

Comment:
This stage is allowed to fail, that needs to be ported too

Reply:
Allow failure/requirements are configured in the branch protection settings of the repo

Reply:
Per default every job is allowed to fail

Reply:
I didn't find it there. Just require pass before merge

Could you share a link?

Reply:
<img width="1024" alt="PNG-Bild" src="https://user-images.githubusercontent.com/120441/71928023-0613b900-3197-11ea-9412-6052be1b936d.png">

Reply:
That's the title I saw, but it's kinda of inversed meaning.
I'll try that

Reply:
I tried to add it somehow